### ISSUE TITLE
The ALabel get messed up if some of its properties get modified in-game

### ISSUE THREAD

**Issue by HanouaJ** [ChemicalInk/godot-arabic-text#6]
I noticed it is very inconvenient to make properties (example: scale, position, outline...) changes to the ALabel during the game. That is because once it is done, the text written in Arabic in the ALabel will get all letters detached from one another. 

In order to illustrate the problem, here is a first example when I have a static non modified ALabel (which works perfectly):
![image](https://user-images.githubusercontent.com/6314530/85208898-26de5e00-b334-11ea-9f80-21e64100bb99.png)

And here is a second example where I try to modify the text's outline's (called also border or stroke) color using the Godot AnimationPlayer:
![image](https://user-images.githubusercontent.com/6314530/85208936-6016ce00-b334-11ea-827d-04e39bbae8e0.png)

A while ago since I came to discover this plugin that you made, I wanted to thank you personally but it is not possible to send you a private message in Github, so I would like to take advantage of this issue to say thank you very much for this precious plugin that made it possible to use Arabic within Godot.

**Comment by HanouaJ** [comment]
bump

**Comment by 3akev** [comment]
Just saw this now, was unavailable.
I'll see what I can do, but honestly, given how little I understand of the codebase, you're probably just as able to investigate as I.
(hint hint, PRs welcome)

**Comment by HanouaJ** [comment]
Tried the latest commit and this bug seems to not exist anymore, closing this issue. However I created a new issue which is a serious one about lines getting inverted.
